Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:2.7.12
+
+COPY ./docker/prod/environment /etc/consul_template_env.sh
+
+COPY ./docker/prod/docker-helpers/install_consul_template.sh \
+     ./docker/prod/docker-helpers/install_runit.sh \
+     /usr/local/bin/
+RUN chmod 755 /usr/local/bin/install_consul_template.sh /usr/local/bin/install_runit.sh && \
+    sync && \
+    install_consul_template.sh && \
+    rm -f \
+        /usr/local/bin/install_consul_template.sh \
+        /usr/local/bin/install_runit.sh
+
+################
+# CAA Redirect #
+################
+
+RUN mkdir /code
+WORKDIR /code
+
+# Python dependencies
+RUN apt-get update && \
+    apt-get install -y \
+            build-essential \
+            libpq-dev \
+            libffi-dev \
+            libssl-dev
+RUN pip install -U cffi
+COPY requirements.txt /code/
+RUN pip install -r requirements.txt
+
+# Node dependencies
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get install -y nodejs
+RUN npm install -g less less-plugin-clean-css
```

Comment:
Yeah, I remember moving those here because of some issue that I'm not seeing anymore. Added *package.json* back.